### PR TITLE
XSIAM IOC Alert

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2644

## Description
Created a new playbook for XSIAM, IOC alert

## Screenshots
IOC_Alert_Sun_Oct_23_2022

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] Only for XSIAM

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 1]
```diff
@@ -0,0 +1,1307 @@
...
+      FileRemediation:
+        complex:
+          root: inputs.FileRemediation
+      HostContainment:
+        simple: "True"
+      UserContainment:
+        simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1037]: Should be here a loop?

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 2]
```diff
@@ -0,0 +1,1307 @@
...
+  value:
+    simple: "False"
+  required: false
+  description: Should we block suspicious/malicious indicators automatically? Specify True/False.
+  playbookInputQuery:
+- key: ShouldCloseAutomatically
+  value:
+    simple: "True"
+  required: false
+  description: Should close the alert automatically if it's established verdict is False Positive?
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1258]: add `Specify True/False.`

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 3]
```diff
@@ -0,0 +1,1307 @@
...
+  playbookInputQuery:
+- key: PreHostContainment
+  value:
+    simple: "False"
+  required: false
+  description: Isolate the host before the investigation phase in case IOC found as suspicious.
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1264]: add `Specify True/False.`

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 4]
```diff
@@ -0,0 +1,1307 @@
...
+  playbookInputQuery:
+- key: ShouldHandleFPautomatically
+  value: {}
+  required: false
+  description: Should we automatically handle false positive alerts? Specify true/false.
+  playbookInputQuery:
+- key: AutoRestoreEndpoint
+  value: {}
+  required: false
+  description: Whether to execute the Recovery playbook.
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1274]: add `Specify True/False.`

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 5]
```diff
@@ -0,0 +1,1307 @@
...
+  required: false
+  description: Isolate the host before the investigation phase in case IOC found as suspicious.
+  playbookInputQuery:
+- key: ShouldHandleFPautomatically
+  value: {}
+  required: false
+  description: Should we automatically handle false positive alerts? Specify true/false.
+  playbookInputQuery:
+- key: AutoRestoreEndpoint
+  value: {}
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1272]: default?

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 6]
```diff
@@ -0,0 +1,1307 @@
...
+  value:
+    simple: Quarantine
+  required: false
+  description: Should be either 'Quarantine' or 'Delete'.
+  playbookInputQuery:
+- key: AutoEradication
+  value:
+    simple: "False"
+  required: false
+  description: |-
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1297]: add `Specify True/False.`

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 7]
```diff
@@ -0,0 +1,1307 @@
...
+        - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: DomainVerdict
+            iscontext: true
+          right:
+            value:
+              simple: Suspicious
+        - operator: isEqualString
```
[the remark this answers is not in the document]
Comment (altmannyarden) [reply Packs/Core/Playbooks/playbook-IOC_Alert.yml:85]: Changed to `includes`

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml (hunk 1 above)
Comment (altmannyarden) [reply Packs/Core/Playbooks/playbook-IOC_Alert.yml:1037]: this is no loop

File: Packs/Core/Playbooks/playbook-IOC_Alert_README.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+* extractIndicators
+* closeInvestigation
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| BlockIndicatorsAutomatically | Should we block suspicious/malicious indicators automatically? Specify True/False. | False | Optional |
+| ShouldCloseAutomatically | Should close the alert automatically if it's established verdict is False Positive? | True | Optional |
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-IOC_Alert_README.md:52]: Can you please re-run the generate-docs so the new changes you added for the inputs' description will be updated?

**Review by ssokolovich — APPROVED** [review]
Looks good :) just check my last comment

**Comment by ShirleyDenkberg** [comment]
@ssokolovich @melamedbn Doc review completed.

**Pull request merged**